src/test/java/org/fundacionjala/sfdc/stepdefinition/ContactSteps.java

```diff
@@ -50,27 +49,27 @@ public void iPressNewContactsButtonANewFormIsDisplayed() {
 
     /**
      * And step.
+     *
      * @param values map.
      */
     @And("^I fill the required information and I press Save$")
-    public void iFillTheRequiredInformationAndIPressSave(final Map<String, String> values) {
+    public void iFillTheRequiredInformationAndIPressSave(final Map<Inputs, String> values) {
         System.out.println("This Step fills account name field");
         contactsNewModifyPage = new SFCNewModifyPage();
-        CommonActions.setValues(values, contactsNewModifyPage.fillMethodsToFields());
+        values.keySet().stream().forEach((step) ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "step" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)